### PR TITLE
Document on the Order Lookup feature

### PR DESCRIPTION
**Pull request by AjinkyaM1**
The document explains the need for this screen and outlines the various types of information it provides within the Fulfillment App.

### CODE REVIEW

File: adoc/flows/SalesOrders/order-lookup.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+description: >-
+  The document highlights the HotWax Commerce Order Lookup feature, which enables store associates to efficiently manage in-store order inquiries with direct access to comprehensive order information.
```
Comment (dt2patel) [inline adoc/flows/SalesOrders/order-lookup.md]:
```suggestion
  The document highlights the HotWax Commerce Order Lookup feature, which enables store associates to manage in-store order inquiries with direct access to comprehensive order information.
```

File: adoc/flows/SalesOrders/OrderApproval.md [hunk 2]
```diff
@@ -5,83 +5,42 @@ description: >-
   customer information and shipping addresses.
 ---
 
-# Order Approval
+# Order Approval Process in ADOC
```
Comment (dt2patel) [inline adoc/flows/SalesOrders/OrderApproval.md]:
```suggestion
# Order Approval Process
```

**Review by dt2patel — CHANGES_REQUESTED** [review]
Please remove Order Approval related changes from this PR.

File: adoc/flows/SalesOrders/order-lookup.md (hunk 1 above)
Comment (AjinkyaM1) [reply adoc/flows/SalesOrders/order-lookup.md]: Removed the order approval changes

**Review by dt2patel — APPROVED** [review]

**Pull request merged**